src/PaymentPart/Output/AbstractOutput.php

```diff
@@ -36,16 +35,16 @@ public function getLanguage(): ?string
         return $this->language;
     }
 
-    public function setPrintable(bool $printable): static
+    public function setPrintOptions(PrintOptions $printOptions): static
     {
-        $this->printable = $printable;
+        $this->printOptions = $printOptions;
 
         return $this;
     }
 
-    public function isPrintable(): bool
+    public function getPrintOptions(): PrintOptions
     {
-        return $this->printable;
+        return $this->printOptions;
     }
```

Comment:
IMO this new feature does not justify a new major release with breaking changes.

The method `setPrintable()` should be kept, but be set to `@deprecated`. It should set `printOptions` to keep the current behaviour.